lib/index.js

```diff
@@ -61,14 +61,17 @@ const PERMISSIONS = bedrock.config.permission.permissions;
  *   tokens, `serviceId` is encoded in the `otpAuthUrl` value returned
  *   by this API.
  * @param [notify=true] `true` to notify the account user, `false` not to.
+ * @param [typeOptions='human'] Setting typeOptions to `human` will generate 9
+ *   character numeric-only nonce and setting it to `machine` will generate
+ *   128-bit bnid.
  *
  * @return a Promise that resolves once the operation completes with optional
  *   token details depending on the type.
  */
 api.set = async ({
   actor, account, email, type, clientId, serviceId, hash,
   authenticationMethod = type, requiredAuthenticationMethods = [],
-  notify = true
+  notify = true, typeOptions = 'human'
```

Comment:
So what I meant here was that `typeOptions` would be an object that would include `entryStyle` such that `typeOptions` could be passed directly into `generateNonce`. Also, I'm not sure we want a default here -- do we? @mattcollier?

```suggestion
  notify = true, typeOptions = {entryStyle: 'human'}
```

Reply:
Updated, but left the default as it is for now. Will update once we get feedback from @mattcollier

Reply:
@mattcollier should we keep the default value here or remove it?

Reply:
Well, the default that is a string certainly needs to change.

DB has discussed this matter of default documentation before.  The issue is that if the default value is not explicit and documented here, one must then dive into the code to discover the relevant defaults are in the constituent API calls.

Then the issue becomes whether or not the default should ONLY be documented on the top level API call or if the default should ALSO be explicitly indicated in code.  I believe that it was agreed at that time that the only way to reason about this is that the top level API call in fact *does* have its own defaults which should be explicit in code and documentation.

@davidlehn I believe you were involved in that discussion, did I summarize properly?  cc: @dlongley

Reply:
@mattcollier I think so.  There are tradeoffs keeping things in sync one way or another.  But yes, easiest way to reason about the system is to have explicit defaults in docs and code.